src/components/Formik/FormikCheckboxInput/FormikCheckboxInput.tsx

```diff
@@ -0,0 +1,43 @@
+import React, {
+  FC,
+  ChangeEvent,
+  FocusEvent,
+} from 'react';
+import CheckboxInput from '../../CheckboxInput/CheckboxInput';
+
+interface FormikCheckboxInputProps {
+  field: {
+    id: string;
+    label: string;
+    name: string;
+    onBlur: (event: FocusEvent<HTMLElement>) => void;
+    onChange: (event: ChangeEvent<HTMLInputElement>) => void;
+    value?: boolean;
+  };
+  form: { [key: string]: any; }; // eslint-disable-line @typescript-eslint/no-explicit-any
+}
+
+const FormikCheckboxInput: FC<FormikCheckboxInputProps> = ({
+  field: {
+    id,
+    label,
+    name,
+    onBlur,
+    onChange,
+    value = false,
+  },
+  form: { touched, errors },
+  ...props
+}) => (
+  <CheckboxInput
+    id={id}
+    label={label}
+    error={touched[name] && errors[name]}
```

Comment:
is it weird we're passing in the whole form object instead of an error directly?

Reply:
@juanfabrega ?

Reply:
@ericyliu the props are passed down via Formik's `Field` component and that component passes down the form. We are documenting their signature in the proptypes to help library users, although in theory, we have no control over those default props (`form` and `field`) being passed down.